engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -825,10 +826,14 @@ case class WorkflowActor(workflow: WorkflowDescriptor)
       case _ => false
     }
 
+    lazy val hasAnythingFailed = executionStore.values exists { x => x == Failed || x == Aborted }
+
+    lazy val workflowFailFastTrigger = workflowFailureMode == NoNewCalls && hasAnythingFailed
```

Comment:
Hmm, I guess I can move the behaviour definition inside the object. The implementation of that behaviour itself though, I think belongs here